src/main/java/org/junitpioneer/jupiter/report/Properties.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.report;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ * <p>Java class for anonymous complex type.
+ *
+ * <p>The following schema fragment specifies the expected content contained within this class.
+ *
+ * <pre>
+ * &lt;complexType&gt;
+ *   &lt;complexContent&gt;
+ *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType"&gt;
+ *       &lt;sequence&gt;
+ *         &lt;element ref="{}property" maxOccurs="unbounded"/&gt;
+ *       &lt;/sequence&gt;
+ *     &lt;/restriction&gt;
+ *   &lt;/complexContent&gt;
+ * &lt;/complexType&gt;
+ * </pre>
+ *
+ *
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = { "property" })
+@XmlRootElement(name = "properties")
+public class Properties {
+
+	@XmlElement(required = true)
+	protected List<Property> property;
```

Comment:
This should be called propertyList, at the very least. Right now it is _very_ confusing.

Reply:
I won't rename the (generated) field, because this would result in an invalid XML (without chaning the XSD).

> Ungültiger Content wurde beginnend mit Element 'propertyList' gefunden. '{property}' wird erwartet.

Translated to english ~

> Invalid content found starting with element 'propertyList' found. '{property}' expected.


And I don't want to rename the element in the XSD / report, but to keep it like this

```
<properties>
    <property name="Issue" value="bishue 135"/>
</properties>

Reply:
Just rename the variable and modify the `propOrder`. You can still satisfy the XSD/Report with explicit naming i.e.: `@XmlElement(required = true, name = "property")`